aws/email.go

```diff
@@ -0,0 +1,75 @@
+package aws
+
+import (
+	"bytes"
+	encsv "encoding/csv"
+	amaws "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ses"
+	c "github.com/companieshouse/innovation-accounts-statistics-tool/config"
+	"github.com/companieshouse/innovation-accounts-statistics-tool/models"
+	"gopkg.in/gomail.v2"
+	"io"
+)
+
+const (
+	subject   = "Small Full Accounts Statistics"
```

Comment:
There's a slight discrepancy here; we're generating a report for CICs who filed small full, but this subject may suggest the numbers are purely for SFA filers?